environment.sh

```diff
@@ -1,17 +1,19 @@
 #!/bin/bash
 ############################################################
-# Help                                                     #
+# Help:                                      #
 ############################################################
 Help()
-{
+{ 
+  # Reset OPTIND so help can be invoked multiple times per shell session.
+  OPTIND=1
    # Display Help
-   echo "Add description of the script functions here."
+   echo "Script to automatically create and validate conda environments."
    echo
-   echo "Syntax: scriptTemplate [-h|t|v]"
+   echo "Syntax: source environment.sh [-h|t|v]"
```

Comment:
I don't know anything about this `Help()` thing, but I don't see `-h` being handled below in the `while getopts` section. Does that handling come for free w/ `Help()`?

Reply:
Its the first option and prints the help options.